src/main/java/seedu/address/storage/JsonAdaptedClient.java

```diff
@@ -76,6 +85,8 @@ public JsonAdaptedClient(Client source) {
         contractExpiryDate = source.getContractExpiryDate().value;
         lastModifiedInstant = source.getLastModifiedInstant().toString();
         tagged.addAll(source.getTags().stream().map(JsonAdaptedTag::new).collect(Collectors.toList()));
+        clientNotes
+                .addAll(source.getClientNotes().stream().map(JsonAdaptedNote::new).collect(Collectors.toSet()));
```

Comment:
These 2 lines are <120 chars, they can be on the same line. Also, breaking it up at the first . makes it inconsistent with the rest of the code base, and harder to read.